### ISSUE TITLE
wallet not syncing 

### ISSUE THREAD

**Issue by earthvisitoruk** [ethereum/mist#3959]
* [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

its been 4 weeks  now and still not syncing  tried transferring with code nothing i think it ripes you off you have not ethereum  in it syncs when you put some in it dont stncs any help please and dont say delete and start again done that about 30 times 
```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

**Comment by LouisWayne** [comment]
The sync issue is really frustrating, please fix this issue ASAP

**Comment by Cleo7** [comment]
I have the same problem and cant log in to my account anywhere :(

**Comment by hughpearse** [comment]
This issue is extremely frustrating, it prevents from making calls like web3.eth.getBalance. The ```--syncmode="light"``` solution is not useful for me, I am conducting research and need the full blockchain.

**Comment by ryanio** [comment]
Hello, we just released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
